seL4-platforms/platforms.yml

```diff
@@ -115,6 +115,16 @@ platforms:
     march: armv8a
     no_hw_test: true
 
+  ROCK3B:
+    arch: arm
+    modes: [64]
+    smp: [64]
+    aarch_hyp: [64]
+    platform: rk3568
+    req: [rk3568]
```

Comment:
This is supposed to be the name of the board in the machine queue (I know, `req` is a terrible name, I just took what we used to have in Bamboo). I don't see a board with that name there yet -- do we know what it will be?

(Should still be fine to merge since we have `no_hw_test: true` for now, just trying to reduce future noise.)

Reply:
Noted, as soon as we have some space in the rack I will put it there.